### PR TITLE
Binance moved to candlestick(only ETH)

### PR DESCRIPTION
**Pull request by alex-subtselnyi**
New Data bound to ETH with candlestick fields

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -52,9 +57,10 @@ ws.on('request', req => {
   });
 });
 
-
-binance.websockets.miniTicker(markets => {
-  let prices = JSON.stringify(markets);
+binance.websockets.candlesticks(['ETHBTC'], "1m", (candlesticks) => {
+  let { e:eventType, E:eventTime, s:symbol, k:ticks } = candlesticks;
```
Comment (PavlyukVadim) [inline index.js]: as these vars are permanent change them to const

File: index.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const Websocket = require('websocket').server;
 const redis = require('redis');
 const client = redis.createClient(); 
 
+//Binance API
 const binance = require('node-binance-api')().options({
   APIKEY: "yTGrczsnJIxGnlRTkmHW2dMAAcPoHNUtoqC2SgT5yWKnv2UFjQ7YaM4zP5zO1z1Q",
```
Comment (PavlyukVadim) [inline index.js]: You should move these keys to separate config file with other settings

Comment (PavlyukVadim) [reply index.js]: And please use single quotes for js

**Review by PavlyukVadim — COMMENTED** [review]
Check code style

File: index.js (hunk 1 above)
Comment (PavlyukVadim) [reply index.js]: koncheniy, tebe esche na A-schku sosat' prepodu dolboeb

Comment (alex-subtselnyi) [reply index.js]: > koncheniy, tebe esche na A-schku sosat' prepodu dolboeb

idi nahui

Comment (alex-subtselnyi) [reply index.js]: > as these vars are permanent change them to const

I will improve code style for you, if you are so strict

File: index.js (hunk 2 above)
Comment (alex-subtselnyi) [reply index.js]: ok

File: index.js (hunk 1 above)
Comment (PavlyukVadim) [reply index.js]: I really appreciate it!

File: index.js [hunk 3]
```diff
@@ -57,10 +59,27 @@ ws.on('request', req => {
...
 });
 
-binance.websockets.candlesticks(['ETHBTC'], "1m", (candlesticks) => {
-  let { e:eventType, E:eventTime, s:symbol, k:ticks } = candlesticks;
-  let { o:open, h:high, l:low, c:close, v:volume, n:trades, i:interval, x:isFinal, q:quoteVolume, V:buyVolume, Q:quoteBuyVolume } = ticks;
-  let prices = JSON.stringify(ticks);
+binance.websockets.candlesticks([CURRENCY], '1m', (candlesticks) => {
+  const { 
+    e:eventType, 
+    E:eventTime,
```
Comment (PavlyukVadim) [inline index.js:65]: please remove useless vars

Comment (alex-subtselnyi) [reply index.js:65]: we will need them later, when we improve our chart. =-|
eventTime - Time when currency was changes, eventType - type of change

Comment (PavlyukVadim) [reply index.js:65]: it's not the best practice, but ok

Comment (alex-subtselnyi) [reply index.js:65]: > it's not the best practice, but ok

omg.....this js-developers....fuck...

**Pull request merged**